modules/lvm2

```diff
@@ -33,6 +33,7 @@ lvm2_configure := \
 	--disable-cache_check_needs_check \
 	--disable-thin_check_needs_check \
 	--with-cluster=none \
+	--with-thin-check= \
```

Comment:
Ha. Confused by this here. Thought I had to hack this one around by config hacks?

Reply:
Not necessarily, the value used in compile-time configuration because the default config value.  You could override it in config later but this is simpler IMO.  In either case, "empty" means skip it per the doc